tasks/src/main/java/org/example/aStartUp2/AStartUp2.java

```diff
@@ -0,0 +1,41 @@
+package org.example.aStartUp2;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+/**
+ * https://codeforces.com/problemset/problem/420/A
+ * Solution with Collections (Set), loops and Scanner
+ * Question:
+ * What is faster to read? A Set (HashSet) with complexity probably O(1) or a List (ArrayList) with complexity probably O(n)?
```

Comment:
o(1), bo jest to zlozonosc liniowa - stala niezaleznie od tego co pobierasz. Nie ma lepszej zlozonosci obliczeniowej od o(1)